client/src/utils/axios-adapter.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+
+class RequestAdapter {
+    constructor(props = {}) {
+        const { baseURL = 'http://localhost:8055/items/todolist', ...rest } = props;
```

Comment:
this should be only URL `http://localhost:8055/`

the url should be located in `.env` files